Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM centos:6.6
 
+RUN \
+    yum install -y 'https://download.postgresql.org/pub/repos/yum/9.5/redhat/rhel-6-x86_64/pgdg-centos95-9.5-3.noarch.rpm'
+
```

Comment:
Why not add this to the yum install below?

Reply:
Makes more sense stylistically to me to keep them all together.

Reply:
I am wondering why not have a single yum install command to avoid creating an extra image layer?

Reply:
That's a good point and I'll fix that now.